src/services/package-manager/package-service.ts

```diff
@@ -0,0 +1,95 @@
+        return new Promise<void>(async resolve => {
+            const fieldsToInclude = ["key", "name", "changeDate", "activatedDraftId", "spaceId"];
+
+            let nodesListToExport: BatchExportNodeTransport[] = (await this.findAllPackages()).map(node => node);
+
+            if (includeDependencies) {
+                fieldsToInclude.push("variables", "type", "value", "dependencies", "id", "version");
+
+                const packagesKeyWithActionFlows = (await packageManagerApi.findAllNodesOfType("SCENARIO")).map(node => node.rootNodeKey);
+                nodesListToExport = nodesListToExport.filter(node => {
+                    return !packagesKeyWithActionFlows.includes(node.rootNodeKey);
+                })
+
+                const variablesByNodeKey = await variableService.getVariablesByNodeKey();
+                nodesListToExport.map(node => {
+                        node.variables = variablesByNodeKey.get(node.key);
+                        return node;
+                    }
+                )
+
+                const draftIdsByNodeId = new Map<string, string>();
+                const nodeIds = nodesListToExport.map(node => {
+                    draftIdsByNodeId.set(node.id, node.workingDraftId);
+                    return node.id;
+                });
+                const dependenciesByPackageId = await this.getPackagesDependenciesByPackageId(nodeIds, draftIdsByNodeId);
+
+                nodesListToExport = nodesListToExport.map(nodeToExport => {
+                    nodeToExport.dependencies = dependenciesByPackageId.get(nodeToExport.id);
+                    return nodeToExport;
+                })
+            }
+            this.exportListOfPackages(nodesListToExport, fieldsToInclude);
+            resolve();
+        })
+    }
+
+    public async getPackagesDependenciesByPackageId(nodeIds: string[], draftIdsByNodeId: Map<string, string>): Promise<Map<string, PackageDependencyTransport[]>> {
+        const dependenciesByPackageKey = new Map<string, PackageDependencyTransport[]>();
+        const packageWithDependencies = await this.getPackagesWithDependencies(nodeIds, draftIdsByNodeId);
+
+        packageWithDependencies.forEach(packageWithDependency => {
+            const dependenciesOfPackage = dependenciesByPackageKey.get(packageWithDependency.rootNodeId) ?? [];
+            dependenciesOfPackage.push(packageWithDependency);
+            dependenciesByPackageKey.set(packageWithDependency.rootNodeId, dependenciesOfPackage);
+        });
+
+        return dependenciesByPackageKey
+    }
+
+    public getPackagesWithDependencies(nodeIds: string[], draftIdsByNodeId: Map<string, string>): Promise<PackageDependencyTransport[]> {
+        return new Promise(async resolve => {
+            const results = await Promise.all(nodeIds.map(async nodeId => {
+                return packageManagerApi.findDependenciesOfPackage(nodeId, draftIdsByNodeId.get(nodeId));
```

Comment:
I think we will need a new API that returns all this information with a single call, but we can work with this for now. Let's add it to the TODO list